qiskit_addon_sqd/subsampling.py

```diff
@@ -32,10 +32,11 @@
 def postselect_and_subsample(
     bitstring_matrix: np.ndarray,
     probabilities: np.ndarray,
-    hamming_right: int,
-    hamming_left: int,
     samples_per_batch: int,
     num_batches: int,
+    *,
```

Comment:
Here and elsewhere, I would keep the same argument ordering (aside from swapping `hamming_right` and `hamming_left`). I think it's fine to err on the side of forcing more keyword arguments. Keeping the ordering will avoid people having to make more changes to their code.